lib/src/main/java/com/smileidentity/SmileID.kt

```diff
@@ -44,6 +62,7 @@ object SmileID {
     var useSandbox: Boolean = true
         private set
 
+    internal var allowOfflineMode: Boolean = false
```

Comment:
if we do it like this, then users can query it too

```suggestion
    var allowOfflineMode: Boolean = false
        private set
```